pphuchar/sonic-buildimage
# [platform/cel] Add fanctrl daemon to support Questone2BD

**- What I did**
Add Celestica fan speed control service to Questone2BD device.
The fan speed control strategy follows Celestica thermal spec.

**- How I did it**
Add a PID fan control application.
Add the PID configuration file under Questone2BD cfg directory.
Add manual fan speed control script of Questone2BD.

**- How to verify it**
Verify that the fanctrl.service is running correctly when OS boot.
Set the fan speed manually to the max and see if the service set the speed back to normal.

**- Description for the changelog**

Add Questone2BD fan speed control application.

**- A picture of a cute animal (not mandatory but encouraged)**
